docs/guide/setup-examples/README.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Introduction
```

Comment:
This should read "Example Setups" and the folder name should reflect the name of this section.